pkg/policy/execute.go

```diff
@@ -203,6 +212,23 @@ func (e *Executor) executeQuery(ctx context.Context, q *Query) (*QueryResult, er
 		if err != nil {
 			return nil, fmt.Errorf("%s: %w", q.Name, err)
 		}
+
+		fmt.Printf("Underlying Type: %+v\n", values)
+		for i, val := range data.FieldDescriptions() {
+			// val.DataTypeOID
+			if values[i] == nil {
+				continue
+			}
+			switch val.DataTypeOID {
+			case 2950:
+				fmt.Printf("val: %v\n", values[i])
+				t := uuid.FromBytesOrNil(ByteSlice(values[i].([16]uint8)))
+				fmt.Printf("Underlying Type: %T, %s\n", t, t.String())
+				values[i] = t
+			}
```

Comment:
I'm not sure if there is a more generic way where we wouldn't have to re-implement for every type